src/main/java/org/ilgcc/app/inputs/Gcc.java

```diff
@@ -191,7 +191,8 @@ public class Gcc extends FlowInputs {
     private String childcareEndTimeSunday;
     private String childAttendsOtherEd;
 
-    private String unearnedIncomeAssetsMoreThanOneMillionDollars;
+  private List<String> unearnedIncomePrograms;
+  private String unearnedIncomeAssetsMoreThanOneMillionDollars;
```

Comment:
Nitpick: make the spacing consistent with the rest of the file

```suggestion
    private List<String> unearnedIncomePrograms;
    private String unearnedIncomeAssetsMoreThanOneMillionDollars;
```